api/serializers.py

```diff
@@ -111,7 +111,7 @@ def partner(request, partner):
 def person(request, person):
     picture = ""
     if person.profile.picture:
-        picture = absolute_url(request, person.profile.picture.url)
+        picture = image_url_or_missing(request, person.profile.picture)
```

Comment:
image_url_or_missing won't ever need to show the missing image since the if-statement guarantees that you have an image when you call it. If I understand the intention correctly, users without an image should show the missing image, then the if-statement should be removed.